src/main/java/com/google/api/codegen/transformer/ruby/RubyGapicSurfaceTransformer.java

```diff
@@ -169,6 +171,27 @@ private ViewModel generateApiClass(GapicInterfaceContext context) {
 
   private ViewModel generateVersionIndexView(Model model, GapicProductConfig productConfig) {
     SurfaceNamer namer = new RubySurfaceNamer(productConfig.getPackageName());
+
+    ImmutableList.Builder<VersionIndexModuleView> moduleViews = ImmutableList.builder();
+    List<String> apiModules = namer.getApiModules();
+    int moduleCount = apiModules.size();
+    for (int i = 0; i < moduleCount; ++i) {
+      if (i == moduleCount - 2) {
```

Comment:
The number `2` is a little magical here. (Presumably, that's 1 module up for the version, plus 1 module up for the product name?)  Some documentation of the reasoning here would be helpful.